lib/matplotlib/tests/test_font_manager.py

```diff
@@ -215,3 +217,41 @@ def test_missing_family(caplog):
         "findfont: Generic family 'sans' not found because none of the "
         "following families were found: this-font-does-not-exist",
     ]
+
+
+def _test_threading():
+    import threading
```

Comment:
Does this test need to check for `dummy_threading` as well?

Reply:
added and importorskip to the main test.